src/app/screens/swap/index.tsx

```diff
@@ -44,10 +41,10 @@ const DownArrowButton = styled.button((props) => ({
 }));
 
 const SendButtonContainer = styled.div((props) => ({
-  paddingBottom: props.theme.spacing(12),
+  paddingBottom: props.theme.spacing(42),
   paddingTop: props.theme.spacing(4),
-  marginLeft: '5%',
-  marginRight: '5%',
+  marginLeft: props.theme.spacing(8),
+  marginRight: props.theme.spacing(4),
```

Comment:
what's the reasoning for not keeping the scrollbar as display:none? it's going to be really brittle if all code which uses ScrollBar has to explicitly allow the right margin.